jobs/slack-statistics.ts

```diff
@@ -20,15 +20,144 @@ export async function postStatisticsToSlack() {
+        },
+    });
+
+    // Pupil screening - success
+    const pupilScreeningSuccessCount = await prisma.pupil_screening.count({
+        where: {
+            createdAt: { gte: begin, lte: end },
+            status: 'success',
+        },
+    });
+
+    // Tutor screening - total
+    const tutorScreeningCount = await prisma.screening.count({
+        where: {
+            createdAt: { gte: begin, lte: end },
+        },
+    });
+
+    // Tutor screening - success
+    const tutorScreeningSuccessCount = await prisma.screening.count({
+        where: {
+            createdAt: { gte: begin, lte: end },
+            success: true,
+        },
+    });
+
+    // Grouping 'knowsCoronaSchoolFrom' - Tutor
+    const tutorKnowsCoronaSchoolFrom = await prisma.screening.groupBy({
+        by: ['knowsCoronaSchoolFrom'],
+        where: {
+            createdAt: { gte: begin, lte: end },
+            knowsCoronaSchoolFrom: { notIn: [''] },
+        },
+        _count: true,
+    });
+
+    let tutorKnowsCoronaSchoolFromTable: [string, string][] = [];
+    tutorKnowsCoronaSchoolFromTable = tutorKnowsCoronaSchoolFrom.map((item) => [item['knowsCoronaSchoolFrom'], item['_count'].toString()]);
+
+    // Instructor screening - total
+    const instructorScreeningCount = await prisma.instructor_screening.count({
+        where: {
+            createdAt: { gte: begin, lte: end },
+        },
+    });
+
+    // Instructor screening - success
+    const instructorScreeningSuccessCount = await prisma.instructor_screening.count({
+        where: {
+            createdAt: { gte: begin, lte: end },
+            success: true,
+        },
+    });
+
+    // Grouping 'knowsCoronaSchoolFrom' - Instructor
+    const instructorKnowsCoronaSchoolFrom = await prisma.instructor_screening.groupBy({
+        by: ['knowsCoronaSchoolFrom'],
+        where: {
+            createdAt: { gte: begin, lte: end },
+            knowsCoronaSchoolFrom: { notIn: [''] },
+        },
+        _count: {
+            createdAt: true,
+        },
+    });
+
+    let instructorKnowsCoronaSchoolFromTable: [string, string][] = [];
+    instructorKnowsCoronaSchoolFromTable = instructorKnowsCoronaSchoolFrom.map((item) => [item['knowsCoronaSchoolFrom'], item['_count'].toString()]);
+
+    // Matches
+    const matchCount = await prisma.match.count({
+        where: {
+            createdAt: { gte: begin, lte: end },
+        },
+    });
+
+    // New subcourses created
+    const subcourseCreatedCount = await prisma.subcourse.count({
+        where: {
+            createdAt: { gte: begin, lte: end },
+        },
+    });
+
+    // Number of free / taken seats of current public subcourses
+    const subcourseSeats = await prisma.subcourse.findMany({
+        select: {
+            course: {
+                select: {
+                    name: true,
+                },
+            },
+            maxParticipants: true,
+            subcourse_participants_pupil: {
```

Comment:
Is it possible to select _count here instead?

Reply:
I tried, and it doesn't seem possible.